integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/BuildApplication.java

```diff
@@ -152,13 +152,30 @@ public static Path buildApplication(Path appSrcPath, Map<String, String> antPara
     } catch (ApiException | IOException  ioex) {
       logger.info("Exception while copying file " + zipFile + " to pod", ioex);
     }
+
     try {
       //copy the build script to /u01 location inside pod
       Kubernetes.copyFileToPod(namespace, webLogicPod.getMetadata().getName(),
           null, BUILD_SCRIPT_SOURCE_PATH, Paths.get("/u01", BUILD_SCRIPT));
     } catch (ApiException | IOException  ioex) {
       logger.info("Exception while copying file " + zipFile + " to pod", ioex);
     }
+    logger.info("kubectl copied " + BUILD_SCRIPT + " into the pod");
```

Comment:
If you have tested it locally and if this code is listing the contents of /u01 successfully then its good to be merged.

Reply:
yes, I did test it locally.